Source/Core/VideoCommon/CPMemory.h

```diff
@@ -255,6 +257,46 @@ struct CPState final
   bool bases_dirty;
   VertexLoaderBase* vertex_loaders[8];
   int last_id;
+
+  CPState & operator=(std::nullptr_t)
```

Comment:
This seems to be the default behavior of aggregate initialization.

Why doesn't e.g.:

```cpp
CPState state;

// ...

state = {};
```

work?

Reply:
yes. You are right.